### ISSUE TITLE
Cannot make it to build vtm-ios-example

### ISSUE THREAD

**Issue by carlos-mg89** [mapsforge/vtm#379]
Hi!

First of all thank you guys for all your work. For Android seems to be working like a charm, and I'm already looking forward to continue and work on my project with it. However, I'm interested in developing my app with support for iOS devices as well.

So far, I've been able to run "gradle build" succesfully at the root of the vtm repo. As in vtm-android-example. It worked as well on vtm-ios-example, after some work to add the robovm plugin into Android Studio.

Then, I launched the vtm-ios-example on XCode, and I'm able to see that project on XCode. I've tried to run it, and see it in the iOS simulator with no luck at all. I get the following error message:

`/Users/Carlos/workspace/vtm/vtm-ios-example/robovm-build/xcode/.ib/Prefix.pch:1:9: 'SVGgh/SVGgh.h' file not found`

I have been loonking in some sites about this issue without any success. I have to admit I'm completely new in the iOS development world, so take that in mind. I have managed to run a simple project on XCode though, in the simulator.

I've followed the instructions here, in order to run the project with RoboVM on XCode:
http://robovm.mobidevelop.com/

And I'm trying with the RoboVM 2.3.1 and 2.3.2 versions. Same results.

**Comment by devemux86** [comment]
cc @Longri

**Comment by devemux86** [comment]
I see the `vtm-ios` uses RoboVM 2.3.1.
@carlos-mg89 have you checked the iOS [documentation](https://github.com/mapsforge/vtm/blob/master/docs/ios.md)?

**Comment by Longri** [comment]
Give me more information please. 
Wich iOS Device or Simmulator? 
Wich RoboVm PlugIn? 
Have you setup your iOS Developer certification? 
I will look at this tomorrow!

**Comment by carlos-mg89** [comment]
@devemux86 true, the vtm-ios uses the 2.3.1, and I'be tried to run the project with it, but seeing it didn't work (and motivated to try a different version than 2.3.1, due to some information that said 2.3.1 wasn't working well with Android Studio 2.3) I wanted to give it a try to RoboVM 2.3.2). 

And yes, I've checked and applied the documentation procedures before opening this issue.

@Longri This is how my environment looks like:
- macOS Sierra 10.12.5
- Xcode 8.3.3
- Not real iOS device, but simulator. iPhone 7 Plus simulated device, with iOS 10.3.1
- RoboVM 2.3.1 (and have tried with 2.3.2 as well)

About "Have you setup your iOS Developer certification?", honestly I don't think I have done that. Is that really necessary to try the app in the simulator?

Thanks to both of you for helping out!

**Comment by Longri** [comment]
No, for run on simulator you don't need a certification.

But you are right, at the moment it have problems with launch on both, Simulator and Device.
I will check this and update all to the latest version.

@devemux86 can you add the Bug label, please.

Another reason to continue working on MOE!

**Comment by carlos-mg89** [comment]
@Longri MOE?

By the way, do you get the same error as I got? I've been asking in the MobiVM/robovm gitter channel, and they haven't been able to provide with a solution either. Not at the moment at least, but we are still discussing it.

**Comment by Longri** [comment]
Not the same but simular. Some recources are missing!

MOE => https://multi-os-engine.org/

**Comment by devemux86** [comment]
Intel Multi-OS Engine, see also [here](http://www.badlogicgames.com/wordpress/?p=3934).

**Comment by carlos-mg89** [comment]
Alright, it seems I made it run in the iOS simulator, thanks to the guys of RoboVM. They said we don't have to run a project on the Xcode, but from the gradle plugin itself. So I executed:
gradle -Probovm.arch=x86_64 -Probovm.device.name=iPhone-5s launchIPhoneSimulator

More info here: https://github.com/MobiVM/robovm/tree/master/plugins/gradle

However, I only see an image of libGDX. Is that what is supposed to load? I was expecting an app similar to the vtm-android-example. A map.
